server/execution/entities/player.py

```diff
@@ -7,14 +7,15 @@
 class Player:
 
     def __init__(self, tan: str, name: str | None, alerted: bool, activation_delay_sec: int, location: Location | None,
-                 accessible_locations: set[Location], role: Role | None = None):
+                 accessible_locations: set[Location], role: Role | None = None, logged_in: bool = False):
```

Comment:
IMO it would make more sense to add a new attribute "status" as an enum which contains the values
- "none"
- "logged in"
- "alerted"

and replaces the "alerted" and "logged_in" attributes.

Reply:
Not sure if it's a good idea to mix these semantics. What if a player is alerted from the start but has not logged in?

And why would you need this one status? I thought we decided on an indicator for "logged-in" and a separate button for "Alert" on the game master execution page?

Reply:
> What if a player is alerted from the start but has not logged in?

That should not be a possibility. Our current flow on the website would be:
1. Create new Execution from Scenario
2. Create player(s)
3. Let player(s) log in
4. Start Execution
5. Alert player(s)

> And why would you need this one status?

Using just one status would ensure that no illegal combinations are possible. E.g. a player can not be not logged in and alerted.

Reply:
One should also be able to load existing Executions from the DB, similar to as it is now.

And why shouldn't a player we alerted from start? In the future a feature could be to resume a simulation from a certain point based on the event log, in which case some players could already be alerted or a scenario were some paramedics are directly at the accident from the start. I don't see why we should limit this artificially.

Reply:
Understood